src/js/server/schemaHandler.js

```diff
@@ -0,0 +1,53 @@
+/*
+
+    An extension of `gpii.express.handler` that adds JSON Schema headers to the outgoing response.
+
+    See this component's documentation for more details:
+
+    https://github.com/the-t-in-rtf/gpii-json-schema/blob/GPII-1336/docs/handler.md
+
+ */
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-express");
+
+fluid.registerNamespace("gpii.schema.handler");
+
+// Send the appropriate headers and then let the underlying grade's `sendResponse` function take over.
+gpii.schema.handler.sendResponse = function (that, response, statusCode, body) {
+    if (!that.options.schemaKey || !that.options.schemaUrl) {
```

Comment:
I have created a new version of the `hasRequiredOptions` grade from `gpii-handlebars`, which is more at home here.  The previous grade was client-only, the new one is usable for both.  I have updated all of the grades with required options to use the new mechanism.  I also added minimal documentation for this, but would prefer to keep it out of the full documentation for the time being.